rst_files/getting_started.rst

```diff
@@ -63,311 +52,52 @@ install the latest version `Anaconda <https://www.anaconda.com/download/#macos>`
+    #. Opening a terminal and typing ``julia`` (Linux)
 
-This downloads, installs, and compiles the correct version of all of packages used in the QuantEcon lectures
-
-Depending on your computer, this may take **10-15 minutes** to run the **first-time**, but be virtually instantaneous thereafter
-
-This code can be put at the top of any notebook in order to get a tested set of
-packages compatible with the code in the QuantEcon notes
-
-More details on packages are explained in a :doc:`later lecture <tools_editors>`
-
-Notebook Basics
-------------------
-
-Notice that in the previous figure the cell is surrounded by a blue border
-
-This means that the cell is selected, and double-clicking will place it in edit mode
-
-As a result, you can type in Julia code and it will appear in the cell
-
-When you're ready to execute these commands, hit ``Shift-Enter``
+Either way you should now be looking at something like this (modulo your operating system)
 
-.. figure:: /_static/figures/nb3_julia.png
+.. figure:: /_static/figures/julia_term_1.png
    :scale: 100%
 
+This is called the JULIA *REPL* (Read-Evaluate-Print-Loop), which we discuss more :ref:`later <julia_repl>`
 
-Modal Editing
-^^^^^^^^^^^^^^^^^^^^
-
-The next thing to understand about the Jupyter notebook is that it uses a *modal* editing system
-
-This means that the effect of typing at the keyboard **depends on which mode you are in**
-
-The two modes are
-
-#. Edit mode
-
-    * Indicated by a green border around one cell, as in the pictures above
-
-    * Whatever you type appears as is in that cell
-
-#. Command mode
-
-    * The green border is replaced by a blue border
-
-    * Key strokes are interpreted as commands --- for example, typing `b` adds a new cell below  the current one
-
-
-(To learn about other commands available in command mode, go to "Keyboard Shortcuts" in the "Help" menu)
-
-
-Switching modes
-^^^^^^^^^^^^^^^^^
-
-* To switch to command mode from edit mode, hit the ``Esc`` key
-
-* To switch to edit mode from command mode, hit ``Enter`` or click in a cell
-
-The modal behavior of the Jupyter notebook is a little tricky at first but very efficient when you get used to it
-
-
-
-Working with Files
-^^^^^^^^^^^^^^^^^^^^^^^^^
-
-To run an existing Julia file using the notebook you can copy and paste the contents into a cell in the notebook
-
-If it's a long file, however, you have the alternative of
-
-#. Saving the file in your **present working directory**
-
-#. Executing ``include("filename")`` in a cell
-
-
-The present working directory can be found by executing the command ``pwd()``
-
-
-
-Plots
-^^^^^^^
-
-Let's generate some plots
-
-First, ensure that you have activated a set of packages within the current Jupyter notebook
-
-.. literalinclude:: /_static/includes/deps_no_using.jl
-
-Now try copying the following into a notebook cell and hit ``Shift-Enter``
+* In the Julia REPL, type (**don't paste**) the following
```

Comment:
This next piece of code is really annoying to type... Could we just say press `]` to enter package mode and then copy the following?

Reply:
Good call. Making that change now.